ts_export/src/type_solver.rs

```diff
@@ -17,7 +17,18 @@ use crate::{
 
 pub struct MemberInfo<'a> {
```

Comment:
Can we also store the original `Field` from syn ? It's available in `serde_derive_internals`, and it would be useful if you need to implement a solver that takes attributes into accounts.